docs/localstorage-disks.md

```diff
@@ -0,0 +1,53 @@
+# Local Storage Placement for VM Disks
+
+
+## Introduction
+
+Virtual Machines are able to have a DataVolume disks that are based on Local Storage PVs. Local Storage PVs are bound to a specific node.
```

Comment:
might be worth noting this is the way things previously worked before the WaitForFirstConsumer fix. reading the introduction, it's a little unclear that we actually solved the problem now. It sounds like an existing problem that we're documenting

Reply:
Right I think I can rename "Introduction" and "Design Overview" sections to "The problem" "The solution"